scout/server/blueprints/cases/templates/cases/case_report.html

```diff
@@ -1156,17 +1156,17 @@ <h6 class="card-subtitle mb-2 text-muted"><span class="badge badge-warning">Part
       </tbody>
     </table>
     <table id="panel-table" class="table table-sm" style="background-color: transparent">
-      <thead>
-        <tr>
-          <th>Affected gene(s)</th>
-          <th>Description</th>
-          <th>Region annotation</th>
-          <th>Transcript - HGVS - Protein</th>
-          <th>OMIM phenotypes [inheritance]</th>
-        </tr>
-      </thead>
-      <tbody>
-        {% if variant.genes %}
+      {% if variant.genes and variant.genes|length <= 5 %}
```

Comment:
```suggestion
      {% if variant.genes and variant.genes|length <= 3 %}
```
I dunno, but it seems about right. If its anywhere above, the right one gets kind of hidden anyway so we would need a way of marking them (or handling them by type, like showing the genomically first and last for INV; prio on panel or disease gene connection aso).

Reply:
We can make it 3! I was unsure myself!